create-env

```diff
@@ -231,6 +230,21 @@ if [ -e \$config_path ]; then
    export XENON_CONFIG=\$config_path
 fi
 
+# grab a local cutax installation if we have one
+MIDWAY_CUTAX_DIR=/dali/lgrandi/xenonnt/software/cutax/v${cutax_version}
```

Comment:
This place is preferable over `/dali/lgrandi/strax/cutax` as you here (`/dali/lgrandi/xenonnt/software/cutax/`) just download each version?

Reply:
Yes I'm going to setup something to automatically put the different cutax versions here, so that it's possible to use different releases. Will work on that today.